worker-taskstowing/src/main/java/com/microfocus/caf/worker/taskstowing/TaskStowingWorker.java

```diff
@@ -0,0 +1,159 @@
+            return createFailureResult(TaskStowingWorkerFailure.FAILED_TO_PARSE_PARTITION_ID_FROM_JOB_TASK_ID);
+        }
+        final String jobId = jobTaskId.getJobId();
+        if (Strings.isNullOrEmpty(jobId)) {
+            LOGGER.error(TaskStowingWorkerFailure.FAILED_TO_PARSE_JOB_ID_FROM_JOB_TASK_ID);
+            return createFailureResult(TaskStowingWorkerFailure.FAILED_TO_PARSE_JOB_ID_FROM_JOB_TASK_ID);
+        }
+
+        try {
+            // workerTask.getContext() should not be null, but it's possible if a worker is passed a message with a context map that
+            // has a key that is not the same as the servicePath, in which case the Worker Framework will pass a null context:
+            // https://github.com/WorkerFramework/worker-framework/blob/develop/worker-core/src/main/java/com/hpe/caf/worker/core/WorkerTaskImpl.java#L124
+            final byte[] contextBytes = workerTaskData.getContext() != null
+                ? workerTaskData.getContext()
+                : OBJECT_MAPPER.writeValueAsBytes(Collections.<String, byte[]>emptyMap());
+            final byte[] trackingInfoBytes = OBJECT_MAPPER.writeValueAsBytes(workerTaskData.getTrackingInfo());
+            final byte[] sourceInfoBytes = OBJECT_MAPPER.writeValueAsBytes(workerTaskData.getSourceInfo());
+
+            databaseClient.insertStowedTask(
+                partitionId,
+                jobId,
+                workerTaskData.getClassifier(),
+                workerTaskData.getVersion(),
+                workerTaskData.getData(),
+                workerTaskData.getStatus().name(),
+                contextBytes,
+                workerTaskData.getTo(),
+                trackingInfoBytes,
+                sourceInfoBytes,
+                workerTaskData.getCorrelationId());
+            return createSuccessResultNoOutputToQueue();
+        } catch (final JsonProcessingException jsonProcessingException) {
+            LOGGER.error(TaskStowingWorkerFailure.FAILED_TO_SERIALIZE_TASK, jsonProcessingException);
+            return createFailureResult(TaskStowingWorkerFailure.FAILED_TO_SERIALIZE_TASK, jsonProcessingException);
+        } catch (final Exception exception) {
+            LOGGER.error(TaskStowingWorkerFailure.FAILED_TO_WRITE_TO_DATABASE, exception);
+            return createFailureResult(TaskStowingWorkerFailure.FAILED_TO_WRITE_TO_DATABASE, exception);
+        }
+    }
+
+    @Override
+    public String getWorkerIdentifier()
+    {
+        return WORKER_IDENTIFIER;
+    }
+
+    @Override
+    public int getWorkerApiVersion()
+    {
+        return WORKER_API_VERSION;
+    }
+
+    @Override
+    public WorkerResponse getGeneralFailureResult(final Throwable t)
+    {
+        return new WorkerResponse(
+            outputQueue,
+            TaskStatus.RESULT_EXCEPTION,
+            WorkerResponseFactory.getExceptionData(t, codec),
+            WORKER_IDENTIFIER,
+            WORKER_API_VERSION,
+            null);
```

Comment:
Minor but for symmetry I feel like you should have a `createExceptionResult()` method in your `WorkerResponseFactory` class for this?